test/errorHandler/ErrorHandlerSpec.scala

```diff
@@ -65,7 +65,7 @@ class ErrorHandlerSpec extends UnitSpec with GuiceOneAppPerSuite with Injecting{
       val notFoundTemplate: Html = errorHandler.notFoundTemplate
       val doc: Document = Jsoup.parse(notFoundTemplate.toString())
 
-      val docTitle = doc.select("title").text()
+      val docTitle = doc.title()
       docTitle should include(messages("title.pattern"))
```

Comment:
Same here please, this should be the output of the `messages("title.pattern")` call, not the call itself

Reply:
fixed